shop3/tests/conditional-tests.lisp

```diff
@@ -0,0 +1,292 @@
+(defpackage #:shop3-conditionals-tests
+  (:use #:cl
+        #:fiveam))
```

Comment:
`(:import-from :shop #:defdomain #:defproblem #:looping-domain)` ?  Just to make this less cumbersome

You might also want `:redefine-ok` kw arguments so that you don't get a big spew of dumb messages when re-running the tests.